freeorion.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
Wrong type `desktop`, according to https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html#spec-appdata-filespec this is legacy and should be replaced with `desktop-application`.

Reply:
Fixed e7cdc82c103aa6d890189fa1e95a6bd8dec53aa5